apps/desktop/native/linux/src/main.rs

```
@@ -0,0 +1,388 @@
+            return Err(zbus::fdo::Error::Failed(format!(
+                "Polkit authorization denied for action: {action_id}"
+            )));
+        }
+
+        Ok(())
+    }
+}
+
+#[interface(name = "com.mycompany.Vpn")]
+impl VpnService {
+    async fn store_config(
```

Clippy (1.97.1)

[warning] this function has too many arguments (8/7)

(warning)